base/docs/helpdb.jl

```diff
@@ -715,9 +715,9 @@ Raises a `SystemError` for `errno` with the descriptive string `sysfunc` if `ift
 systemerror
 
 doc"""
-    writedlm(f, A, delim='\\t')
+    writedlm(f, A, delim='\\t', header_string=nothing)
```

Comment:
need to adjust the rst prototype as well, then run `./julia doc/genstdlib.jl`

generally rebases are preferred vs merges